### PR TITLE
Fix the not translated area

### PR DESCRIPTION
**Pull request by mizphses**
#1345

### CODE REVIEW

**Review by septs — APPROVED** [review]

**Review by Tedko — APPROVED** [review]

File: src/_locales/ja/messages.json [hunk 1]
```diff
@@ -260,28 +260,28 @@
...
+    "plugin_file_service_file_key": "ファイルキー:",
+    "plugin_file_service_unencrypted": "このファイルは暗号化されていません",
+    "plugin_file_service_signing_failed": "サービスが使用できません。今しばらくお待ちください。",
+    "plugin_file_service_on_encrypt_it": "暗号化しよう",
+    "plugin_file_service_on_show_more": "サポートされていません",
+    "plugin_file_service_legal_text": "このプラグインを使用することで<terms>利用規約</terms>並びに<policy>個人情報保護方針（プライバシーポリシー）</policy>に同意したこととなります。",
+    "plugin_file_service_legal_terms_link": "https://legal.maskbook.com/arweave/file-service/plugin-terms.html",
+    "plugin_file_service_legal_policy_link": "https://legal.maskbook.com/arweave/file-service/privacy-policy-uploader.html",
+    "plugin_file_service_drop_indicator": "ファイルをドラッグ＆ドロップしてアップロード！",
+    "plugin_file_service_drop_hint_1": "ファイル容量の超過：{{limit}}.",
```
Comment (septs) [inline src/_locales/ja/messages.json:281]: the `：` is full-width symbol,
but `plugin_file_service_display_name` and `plugin_file_service_file_key` is half-width symbol 

why?

Comment (septs) [reply src/_locales/ja/messages.json:281]: @mizphses thanks.

Comment (mizphses) [reply src/_locales/ja/messages.json:281]: It's might be because of the IME's conversion... I should check before commit.

Comment (septs) [reply src/_locales/ja/messages.json:281]: @mizphses the case, use half-width symbol? or full-width symbol?
(i will fix it myself)

**Pull request merged**